### ISSUE TITLE
Scrolling causes drawerView to be displayed

### ISSUE THREAD

**Issue by georgemp** [gloubibou/HHPanningTableViewCell#26]
On the demo project (in iOS 7), scroll all the way to the bottom and then back up again. In a few times of doing this, I start seeing the drawerView displayed on some rows that are newly displayed as they scroll by. I am not sure what is triggerring this. I see this a lot more often (don't have to scroll as much) in my app, but, am able to trigger this in the demo app as well. Any ideas? Thanks

**Comment by gloubibou** [comment]
Sounds like a bug in prepareForReuse. The drawer should be reset here

**Comment by georgemp** [comment]
It doesn't seem to be an issue with prepareForReuse. As part of debugging, I tried commenting out the deque line so, that new cells are always initialized. Added log statements to prepareForReuse and init, to confirm while running. The only log statements seen are init statements at this point. The issue still persists after scrolling up and down a few times. Also,
1. Other cells can be panned across to reveal the drawer on them. The drawer on the cell that was revealed on account of scrolling does not close.
2. Scroll back up and down so that the cell (whose drawer view was revealed on account of the scroll) disappears and reappears. Tapping on the cell causes a crash (EXC_BAD_ACCESS) at times.
